src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,39 @@
+.catalog {
+  max-width: 944px;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    display: grid;
+    grid-template-columns: repeat(2, $cardWidth);
+    gap: 48px;
+    justify-content: center;
```

remove the grid and gap from @media and write it in default .catalog { 
}
```suggestion
    grid-template-columns: repeat(2, $cardWidth);
```